feat (docs): [general] Remove `Preview` since CA is now GA(ed) (#25)
mspnp/container-apps-fabrikam-dronedelivery: README.md

      ingestionQueueName=$INGESTION_QUEUE_NAME
   ```

>    :eyes: Please note that Azure Container Apps as well as this ARM API specification are currently with [limited `location` support](https://azure.microsoft.com/global-infrastructure/services/?products=container-apps).

Reviewer: ```suggestion
   :eyes: Please note that Azure Container Apps as well as this ARM API specification currently have [limited `location` support](https://azure.microsoft.com/global-infrastructure/services/?products=container-apps).
```
